index.bs

```diff
@@ -361,7 +361,70 @@ Report long tasks {#report-long-tasks}
 Security & privacy considerations {#priv-sec}
 ===============================================
 
-Long Tasks API adheres to the same-origin policy by including origin-safe attribution information about the source of the long task.
-There is a 50ms threshold for long tasks. Together this provides adequate protection against security attacks against browser.
+Long Tasks API adheres to the same-origin policy by including origin-safe attribution information about
+the source of the long task. There is a 50ms threshold for long tasks. Together this provides adequate
+protection against security attacks against the browser itself.
+
+The Long Tasks API provides timing information about the duration and type of tasks executed by the user,
+as well as attribution such as source code location for function calls. This could enable an attacker
+to perform side-channel timing attacks to guess user’s action, or identify the user. For example, a pattern of
+long script followed by a long render could be put together to guess user’s interaction with a social
+widget. Detailed function call attribution would be used to determine the user’s action.
+
+However, privacy related attacks are possible. While the API doesn’t introduce any new privacy attacks,
```

Comment:
do you mean that privacy related attacks are already possible by other means?

Reply:
Removed sentence, I think it is clearer without this.